domain/getter_header.go

```diff
@@ -1,19 +1,19 @@
-package pkg
+package domain
 
 import (
 	"net/http"
 
-	"github.com/LUSHDigital/litmus/format"
 	"github.com/pkg/errors"
+	"strings"
 )
 
 // HeaderGetter extracts information from response headers.
 type HeaderGetter struct{}
 
 // Get extracts a value out of request headers.
-func (e *HeaderGetter) Get(c format.GetterConfig, header http.Header) (value string, err error) {
+func (e *HeaderGetter) Get(path string, header http.Header) (value string, err error) {
 	for k, v := range header {
-		if c.Path == k && len(v) > 0 {
+		if strings.ToLower(path) == strings.ToLower(k) && len(v) > 0 {
```

Comment:
You can use `strings.EqualFold` here instead.

Reply:
good shout, done